Sdg23 add events page (#450)
MDAnalysis/MDAnalysis.github.io: pages/events.md

---
layout: page
title: Events
order: 4
---

> MDAnalysis hosts and participates in various **community-driven events, user group meetings (UGMs), and workshops** to foster collaboration, learning, and to contribute to the MDAnalysis ecosystem.

Reviewer: ```suggestion
MDAnalysis hosts and participates in various **community-driven events, user group meetings (UGMs), and workshops** to foster collaboration and learning.
```

> This page provides information on **upcoming and past events**, including **UGMs, technical workshops, and training sessions**. Whether you're a developer, researcher, or new to the field, these events provide opportunities to **engage, learn, and contribute**.

Reviewer: ```suggestion
This page provides information on **upcoming and past events**, including **UGMs, workshops and talks**. Whether you're a developer, researcher, or new to the field, these events provide opportunities to engage with and contribute to the MDAnalysis ecosystem.
```

> For mentoring programs and outreach opportunities, please visit the [Contribute]({{ site.baseurl }}/pages/contribute/) page.

Reviewer: not sure this is needed but I'll defer to other people's opinion

Reviewer: @jennaswa would you mind having a look?

<!--  Uncomment the following section when new events are announced 

## Upcoming Events

- [Upcoming Event Name](#)
-->

## User Group Meetings (UGMs)

> UGMs are community-driven meetings where developers and users present new features, share use cases, and discuss improvements.

Reviewer: ```suggestion
UGMs are meetings where users meet developers, share best practices, present new features, and discuss improvements.
```

### Past UGMs
- [UGM 2024][]
- [UGM 2023][]

## Workshops & Training

> MDAnalysis organizes workshops and training sessions to help users and developers improve their skills in molecular dynamics analysis.

Reviewer: it may be worth to add that workshops target both new/beginner and intermediate users.
todo (for myself): we must add a link to workshop materials.

Author: Added to the TODO list in [here](https://docs.google.com/document/d/1vh2puLFjEF61q-FrNlJlw2pFSeGIPjlN/edit).

> Recordings of many past workshops are available on our [YouTube channel][].

Reviewer: ```suggestion
Recordings of past workshops are available on our [YouTube channel][].
```

---
layout: page
title: Events
order: 4
---

MDAnalysis hosts and participates in various **community-driven events, user group meetings (UGMs), and workshops** to foster collaboration and learning.

This page provides information on **upcoming and past events**, including **UGMs, workshops and talks**. Whether you're a developer, researcher, or new to the field, these events provide opportunities to engage with and contribute to the MDAnalysis ecosystem.

For mentoring programs and outreach opportunities, please visit the [Contribute]({{ site.baseurl }}/pages/contribute/) page.

<!--  Uncomment the following section when new events are announced 

## Upcoming Events

- [Upcoming Event Name](#)
-->

## User Group Meetings (UGMs)

> UGMs are meetings where users meet developers, share best practices, present new features, and discuss improvements.

Reviewer 2: ```suggestion
UGMs are meetings where users meet developers, share best practices, present new features, and discuss improvements. If you are interested in sponsoring or potentially hosting a UGM (or similar MDAnalysis event), please [let us know](mailto: ugm@mdanalysis.org).
```

Reviewer 2: I'm not sure if others agree about adding this, but I wonder if we should start creating communication channels for people to express interest in helping organize future events (and finding potential funding sources for them).

Reviewer: I think it's good to add - committing your suggestion!

### Past UGMs
- [UGM 2024][]
- [UGM 2023][]

## Workshops & Training

MDAnalysis organizes workshops and training sessions designed for both beginner and intermediate users to help them improve their skills in molecular dynamics analysis.

<!-- TODO: Add a link to workshop materials when available -->

Recordings of past workshops are available on our [YouTube channel][].

> ### Past Workshops

Reviewer 2: I already added suggested code for this (see https://github.com/MDAnalysis/MDAnalysis.github.io/pull/450#discussion_r1995469747). Any reason not to just commit that with this PR?

---
layout: page
title: Events
order: 4
---

MDAnalysis hosts and participates in various **community-driven events, user group meetings (UGMs), and workshops** to foster collaboration and learning.

This page provides information on **upcoming and past events**, including **UGMs, workshops, and talks**. Whether you're a developer, researcher, or new to the field, these events provide opportunities to engage with and contribute to the MDAnalysis ecosystem.

For the latest discussions, **join the MDAnalysis community channels** by visiting our [Community]({{ site.baseurl }}/pages/community/) page.

For mentoring programs and outreach opportunities, please visit the [Contribute]({{ site.baseurl }}/pages/contribute/) page.

<!--  Uncomment the following section when new events are announced 

## Upcoming Events

> - [Upcoming Event Name](#)

Reviewer 3: This could be an automated piece of code, based on tags. (Future work.)

-->

## User Group Meetings (UGMs)

UGMs are meetings where users meet developers, share best practices, present new features, and discuss improvements. If you are interested in sponsoring or potentially hosting a UGM (or similar MDAnalysis event), please [let us know](mailto: ugm@mdanalysis.org).

### Past UGMs

<div style="display: flex; align-items: center; gap: 10px;">
  <img src="/public/images/mdanalysis-ugm2024.png" title="MDAnalysis UGM 2024 Logo" 
       alt="MDAnalysis UGM 2024 Logo" style="height: 5em;" />
  <p>
    <strong><a href="https://www.mdanalysis.org/pages/ugm2024/">UGM 2024</a></strong>

>     (<a href="https://www.mdanalysis.org/2024/09/11/mda_2024events/#mdanalysis-ugm-user-group-meeting">Post-UGM Summary</a>)

Reviewer 3: Use `{% post_url %}` for internal site links to posts

  </p>
</div>

<div style="display: flex; align-items: center; gap: 10px;">
  <img src="/public/images/ugm.jpeg" title="MDAnalysis UGM 2023 Logo" 
       alt="MDAnalysis UGM 2023 Logo" style="height: 5em;" />
  <p>
    <strong><a href="https://www.mdanalysis.org/pages/ugm2023/">UGM 2023</a></strong>

>     (<a href="https://www.mdanalysis.org/2024/01/15/mda_events/#mdanalysis-ugm-user-group-meeting">Post-UGM Summary</a>)

Reviewer 3: Use `{% post_url %}` for internal site links to posts

  </p>
</div>

## Workshops & Training

MDAnalysis organizes workshops and training sessions designed for beginner, intermediate, and advanced users to help them improve their skills in molecular dynamics analysis and optimize their experience with MDAnalysis.

<!-- TODO: Add a link to workshop materials when available -->

Workshop materials are made publicly available under open source licenses. Recordings of [past workshops](#past-workshops) are available on our [YouTube channel][].

### Past Workshops

| **Title** | **Date(s)** | **Format** | **Relevant Links** |
| ----- | ------- | ------ | -------------- |

> | MDAnalysis Streaming Workshop | Dec 4, 2024 | Online | [Announcement](https://www.mdanalysis.org/2024/11/03/ASU_streaming_workshop/), [Materials](https://github.com/MDAnalysis/imd-workshop-2024), [Recording](https://www.youtube.com/watch?v=fjBTvnEADGs) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | MDAnalysis Intermediate Workshop, CCPBioSim Training Week | Sep 5, 2024 | Hybrid | [Announcement](https://www.ccpbiosim.ac.uk/training2024), [Materials](https://github.com/MDAnalysis/MDAnalysisWorkshop-Intermediate1Day), [Post-Workshop Summary](https://www.mdanalysis.org/2024/09/11/mda_2024events/#september-5th-mdanalysis-intermediate-workshop-ccpbiosim-training-week) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | Moving from User to Developer: Analyzing Molecular Simulations and Building New Tools | Jun 24-25, 2024 | Hybrid | [Announcement](https://www.mdanalysis.org/2024/05/14/ASUworkshop/), [Materials](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/tree/jun24-ws), [Post-Workshop Summary](https://www.mdanalysis.org/2024/09/11/mda_2024events/#june-24th-25th-moving-from-user-to-developer-analyzing-molecular-simulations-and-building-new-tools) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | Introduction to Molecular Dynamics Trajectory Analysis using MDAnalysis | May 10, 2024 | Hybrid | [Announcement](https://www.mdanalysis.org/2024/03/30/tyc-workshop/), [Materials](https://github.com/MDAnalysis/MDAnalysisWorkshop-Intro1Day/tree/may24-ws), [Recording](https://youtu.be/p3OUUnHXQjU), [Post-Workshop Summary](https://www.mdanalysis.org/2024/09/11/mda_2024events/#may-10th-introduction-to-molecular-dynamics-trajectory-analysis-using-mdanalysis) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | Introduction to MDAnalysis and Molecular Nodes | Feb 28, 2024 | Online | [Announcement](https://www.mdanalysis.org/2024/02/05/mdaMNworkshop/), [Materials](https://github.com/MDAnalysis/MDAnalysisWorkshop-Intro0.5Day/tree/feb24-ws), [Recording](https://youtu.be/3zKBjnRnAMg), [Post-Workshop Summary](https://www.mdanalysis.org/2024/09/11/mda_2024events/#februrary-28th-introduction-to-mdanalysis-and-molecular-nodes-online) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | Intro to MDAnalysis Workshop | Oct 25, 2023 | Online | [Announcement](https://www.mdanalysis.org/2023/08/11/workshops/), [Materials](https://github.com/MDAnalysis/MDAnalysisWorkshop-Intro0.5Day/tree/oct23-ws), [Recording](https://youtu.be/njzoNzOwR78), [Post-Workshop Summary](https://www.mdanalysis.org/2024/01/15/mda_events/#mdanalysis-online-training-workshops) |

Reviewer 3: Use {% post_url %} for internal site links to posts

| MDAnalysis Workshop at CCP5 Summer School | Jul 26-27, 2022 | In-Person| [Website](https://summer2022.ccp5.ac.uk/) |

> | MDAnalysis and Machine Learning for Molecular Simulations | Jun 9-10, 2022 | Hybrid | [Materials](https://github.com/MDAnalysis/WorkshopMDMLEdinburgh2022), [Post-Workshop Summary](https://www.mdanalysis.org/2022/07/19/UK_workshops/) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | Using MDAnalysis for Efficient Simulation Pre- and Post-Processing @SURF | May 26-28, 2021 | Online | [Announcement](https://www.mdanalysis.org/2021/04/09/prace-workshop/), [Materials](https://github.com/MDAnalysis/WorkshopPrace2021) |

Reviewer 3: Use {% post_url %} for internal site links to posts

> | MDAnalysis 2018 Workshop and Hackathon at Northwestern University | Nov 12-13, 2018 | In-Person | [Announcement](https://www.mdanalysis.org/2018/08/24/numfocus-workshop/), [Materials](https://github.com/MDAnalysis/WorkshopHackathon2018) | 

Reviewer 3: Use {% post_url %} for internal site links to posts

## Hosting Your Own MDAnalysis Workshops
**Have you previously hosted/Are you already organizing an MDAnalysis workshop?** [Let us know][partner-workshops], as we would love to help you spread the word and share information about your event.

**Would you like to host your own MDAnalysis workshop**, but don't know where to get started, have questions on available workshop materials, or would like to seek other support from the [MDAnalysis workshop team]({{ site.baseurl }}/pages/team/)? [Get in touch][partner-workshops] to let us know how we can best support you.

[UGM 2024]: https://www.mdanalysis.org/pages/ugm2024/
[UGM 2023]: https://www.mdanalysis.org/pages/ugm2023/

> [2024 Events Recap]: https://www.mdanalysis.org/2024/09/11/mda_2024events/

Reviewer 3: Use {% post_url %} for internal site links to posts

> [2023 Events Recap]: https://www.mdanalysis.org/2024/01/15/mda_events/

Reviewer 3: Use {% post_url %} for internal site links to posts

> [YouTube channel]: https://www.youtube.com/@mdanalysis3040

Reviewer 3: might be worthwhile making it a `_config.yml` variable
